test/block.js

```diff
@@ -30,140 +22,115 @@ function once (fn) {
 // 2. alice never tries to connect to bob. (removed from peers)
 // 3. carol will not give bob any, she will not give him any data from alice.
 
-function seed (name) {
-  return crypto.createHash('sha256').update(name).digest()
-}
+const CONNECTION_TIMEOUT = 500
```

Comment:
I'm always confused by these timeout, are they ms or s? So I often postfix them with // ms if that is the case. What do you think?